site/content/docs/developing/internal-albs.en.md

```diff
@@ -29,9 +29,6 @@ network:
     placement: private
 ```
 
-!!!attention
-    Currently, you must use a new Backend Service that has not yet been deployed. Very soon, you will be able to add internal load balancers to existing Backend Services!
-
```

Comment:
actually....this may still be true 🤔 did you try it?

Reply:
Ahh, yeah-- you're right. I thought we had added this box bc we were waiting for env manifests. Reinstating!